plugin/db/mongodb/create_index_on_migrationHistory_collection_command.json

```diff
@@ -0,0 +1,37 @@
+{
+  "createIndexes": "migrationHistory",
+  "indexes": [
+    {
+      "key": {
+        "namespace": 1,
+        "sequence": 1
```

Comment:
Need to check the query pattern to choose using ASC or DESC index. Save for bytebase_idx_unique_migration_history_namespace_version and bytebase_idx_migration_history_namespace_created

Reply:
ASC or DESC index in the compound index can help the engine to sort the records quickly, but it seems that we don't need to sort by these columns.